server/routerlicious/packages/lambdas/src/scribe/lambda.ts

```diff
@@ -367,6 +370,7 @@ export class ScribeLambda implements IPartitionLambda {
 									this.serviceConfiguration.scribe.clearCacheAfterServiceSummary,
 								);
 								this.updateLastSummarySequenceNumber(operation.sequenceNumber);
+								this.updateValidParentSummaries(summaryResponse);
```

Comment:
Do we not need to do this in the `writeClientSummary()` scenario, on line 279? Or do we only care about updating the list of valid parent summaries after a service summary? Isn't the client summary also a valid parent summary that should be included in the list?

Reply:
The last client summary is already factored into the summaryWriter's summary check (see line 114 of summaryWriter.ts, `lastSummaryHead !== content.head`). The checkpoint also already tracks latest client summary in `lastClientSummaryHead` so there is no need to also track it in validParentSummaries.